### PR TITLE
[MS-SQL] Add Support for multiple SSL Certificates

### PR DESCRIPTION
**Pull request by moxarth-elastic**
## Closes https://github.com/elastic/connectors-python/issues/1111

This PR includes:
- Add multiple certificates in `get_pem_format`
- Add condition to avoid unnecessary logger while connector founds tables in database.

#### Pre-Review Checklist
- [x] this PR has a meaningful title
- [x] this PR links to all relevant github issues that it fixes or partially addresses
- [x] this PR has a thorough description
- [x] Covered the changes with automated tests
- [x] Tested the changes locally

### CODE REVIEW

File: connectors/sources/generic_database.py [hunk 1]
```diff
@@ -447,17 +447,18 @@ async def fetch_rows(self, schema=None):
         """
         tables_to_fetch = await self.get_tables_to_fetch(schema)
         if self.database in self.tables_to_skip.keys():
-            tables_to_fetch = [
-                *(set(tables_to_fetch) - set(self.tables_to_skip[self.database]))
-            ]
+            tables_to_fetch = set(tables_to_fetch) - set(
+                self.tables_to_skip[self.database]
+            )
+        tables_to_fetch = list(tables_to_fetch)
```
Comment (artem-shelkovnikov) [inline connectors/sources/generic_database.py:453]: Why change this?

File: connectors/sources/mssql.py [hunk 2]
```diff
@@ -143,9 +142,28 @@ async def close(self):
...
+            key = key.replace(" ", "\n")
+            key = " ".join(key.split("\n", max_split))
+            key = " ".join(key.rsplit("\n", max_split))
+            all_cert += key + "\n"
+        return all_cert
+
     def create_pem_file(self):
         """Create pem file for SSL Verification"""
-        pem_format = get_pem_format(key=self.ssl_ca, max_split=1)
+        pem_format = self.get_pem_format(keys=self.ssl_ca, max_split=1)
```
Comment (artem-shelkovnikov) [inline connectors/sources/mssql.py:166]:
```suggestion
        pem_certificates = self.get_pem_format(keys=self.ssl_ca, max_split=1)
```

File: connectors/sources/mssql.py [hunk 3]
```diff
@@ -143,9 +142,28 @@ async def close(self):
             return
         self.connection.close()
 
+    def get_pem_format(self, keys, max_split=-1):
```
Comment (artem-shelkovnikov) [inline connectors/sources/mssql.py:145]: Why create a new function instead of updating the old one?

Comment (moxarth-elastic) [reply connectors/sources/mssql.py:145]: As customers are waiting for MS-SQL, we have updated it at the connector level so other connectors do not get affected. Also, we are working on it. Once testing is done for all the connectors, we will raise a follow-up PR in which we move these changes to utils and remove them from MS-SQL.

File: connectors/sources/generic_database.py (hunk 1 above)
Comment (moxarth-elastic) [reply connectors/sources/generic_database.py:453]: As type of `tables_to_fetch` is map, we need to convert it to list. also to remove multiple type conversion we remove it from the L450

**Review by artem-shelkovnikov — APPROVED** [review]

**Pull request merged**